index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title></title>
+  <link rel="stylesheet" href="stylesheets/style.css">
+  <link rel="stylesheet" href="stylesheets/index.css">
+</head>
+<body>
+  <header>
+    <div id="header">
+      <h1>The ADA Hall of Fame</h1>
+    </div>
+    <div>
+      <ul class="main_page_links">
+          <li><a href="./pages/education.html">Education</a></li>
+          <li><a href="./pages/hobbies.html">Hobbies</a></li>
+          <li><a href="./pages/fun_facts.html">Fun Facts</a></li>
+      </ul>
+    </div>
+    
+  </header>
+  <main>
+    <section class="instructors">
+      <nav class="nav_adrian">
```

Comment:
`nav` tags aren't usually used in this sense, this feels more like a `section` to me? `nav` tags are for links that help the user navigate to different pages on the sites.